<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_imagen_original">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>imagen original</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagen_procesada">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>imagen procesada</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1001</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}</string>
   </property>
   <property name="text">
    <string>                                                                                              PROCESAMIENTO DE IMAGENES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Heavy</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}
</string>
   </property>
   <property name="text">
    <string>  Espacio de color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_cargar_imagen">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}
</string>
   </property>
   <property name="text">
    <string>Cargar imagen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_espacio_color">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string extracomment="BGR"/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_aplicar_color">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Heavy</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}</string>
   </property>
   <property name="text">
    <string>Aplicar cambio de espacio de color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_ecualizar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Heavy</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}</string>
   </property>
   <property name="text">
    <string>Ecualizar histograma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_binarizar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}</string>
   </property>
   <property name="text">
    <string>Binarizar Imagen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_morfologia">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}</string>
   </property>
   <property name="text">
    <string>Aplicar morfología</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_contar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}</string>
   </property>
   <property name="text">
    <string>Contar células</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_filtro_extra">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}
</string>
   </property>
   <property name="text">
    <string>Metodo extra (filtro bilateral)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_reiniciar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}
</string>
   </property>
   <property name="text">
    <string>Restaurar imagen original</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_volver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Heavy</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}</string>
   </property>
   <property name="text">
    <string>Volver al menu principal</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_umbral">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Heavy</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0,
                                      stop:0 #ECECEC, stop:1 #DADDE1);
    color: rgba(50, 50, 50, 220);
    border: 1px solid #B0B0B0;
    border-radius: 5px;
    padding: 2px 6px;
  font-family: &quot;Franklin Gothic Heavy&quot;;
    font-size: 13pt;
}



/* Flechas internas */
QSpinBox::up-arrow, QSpinBox::down-arrow {
    width: 10px;
    height: 10px;
}

/* Al pasar el mouse por los botones */
QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #C7C9CD;
}
</string>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="prefix">
    <string comment="UMBRAL">Umbral: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_kernel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0,
                                      stop:0 #ECECEC, stop:1 #DADDE1);
    color: rgba(50, 50, 50, 220);
    border: 1px solid #B0B0B0;
    border-radius: 5px;
    padding: 2px 6px;
  font-family: &quot;Franklin Gothic Heavy&quot;;
    font-size: 13pt;
}



/* Flechas internas */
QSpinBox::up-arrow, QSpinBox::down-arrow {
    width: 10px;
    height: 10px;
}

/* Al pasar el mouse por los botones */
QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #C7C9CD;
}
</string>
   </property>
   <property name="prefix">
    <string>Kernel: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_morfologia">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string comment="APlicación"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultado_celulas">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_guardar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #34598a; 
    color: white;
    border: none;
    padding: 10px 20px; 
    font-size: 16px; 
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #2a5578f;  
}
QPushButton:pressed {
    background-color: #2a5578; 
}
</string>
   </property>
   <property name="text">
    <string>GUARDAR EN BASE DE DATOS </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1011</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/QS.jpg);
background-position: center;
background-repeat: no-repeat;
background-size:contain; /* o prueba &quot;contain&quot; si quieres que no recorte */
background-color: rgba(255, 255, 255, 0.8); /* más opaco */
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_imagen_original</zorder>
  <zorder>label_imagen_procesada</zorder>
  <zorder>label</zorder>
  <zorder>boton_cargar_imagen</zorder>
  <zorder>combo_espacio_color</zorder>
  <zorder>boton_aplicar_color</zorder>
  <zorder>boton_ecualizar</zorder>
  <zorder>boton_binarizar</zorder>
  <zorder>label_3</zorder>
  <zorder>boton_morfologia</zorder>
  <zorder>boton_contar</zorder>
  <zorder>boton_filtro_extra</zorder>
  <zorder>boton_reiniciar</zorder>
  <zorder>boton_volver</zorder>
  <zorder>spin_umbral</zorder>
  <zorder>label_4</zorder>
  <zorder>spin_kernel</zorder>
  <zorder>label_5</zorder>
  <zorder>combo_morfologia</zorder>
  <zorder>label_resultado_celulas</zorder>
  <zorder>boton_guardar</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
